storage-proofs/porep/Cargo.toml

```diff
@@ -9,34 +9,36 @@ repository = "https://github.com/filecoin-project/rust-fil-proofs"
 readme = "README.md"
 
 [dependencies]
+anyhow = "1.0.23"
```

Comment:
can you please not reorder the deps, this is gonna be painful when I have to rebase some PRs onto this

Reply:
Fine, but let's reorder them at some point in a single focused PR, then keep them that way.

Reply:
fair enough